lib/SILOptimizer/Transforms/CopyPropagation.cpp

```diff
@@ -35,6 +35,16 @@
 /// TODO: Cleanup the resulting SIL by deleting instructions that produce dead
 /// values (after removing its copies).
 ///
+/// PASS DEPENDENCIES:
+/// - ComputeSideEffects
+///
+/// ANALYSES USED:
+/// - BasicCalleeAnalysis
+/// - DeadEndBlocksAnalysis
+/// - DominanceAnalysis
+/// - NonLocalAccessBlockAnalysis
+/// - PostOrderAnalysis
```

Comment:
Awesome! Maybe we can get pass authors to start listing dependencies. I sometimes even include *why* the dependence exists.